vissl/hooks/wandb_hook.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import logging
+
+import torch
+from classy_vision import tasks
+from classy_vision.generic.distributed_util import is_primary
+from classy_vision.hooks.classy_hook import ClassyHook
+
+if is_primary():
```

Comment:
why do this check? Importing on all ranks should be fine too?

Reply:
wandb import has some overhead to it, so I was trying to limit it to the main worker since it's the only one using it. This can be changed

Reply:
I think it makes sense to me to do it on primary_rank only. :)